packages/ws/src/utils/constants.ts

```diff
@@ -18,13 +18,19 @@ export enum Encoding {
  * Valid compression methods
  */
 export enum CompressionMethod {
-	ZlibStream = 'zlib-stream',
+	ZLibNative,
```

Comment:
Why the capitalized `L` 😢

Reply:
dunno. felt right to me. if you want, we can change it back

Reply:
We settled on `Zlib`, vote was here:

- https://canary.discord.com/channels/222078108977594368/1209960895141388389/1236249988871229461